friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+const Container = styled.div`
+    align-items: center;
+    margin: 0 auto;
+    padding-top: 20px;
+    max-width: 400px;
+    width: 100%;
+`;
+
+const ButtonContainer = styled.div`
+    display: flex;
+    justify-content: space-evenly;
+`;
+
+class FriendForm extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: '',
+            age: '',
+            email: '',
+        }
+    }
+
+    changeHandler = (event) => {
+        this.setState({
+            [event.target.name] : event.target.value
+        })
+    }
+
+    submitHandler = (event) => {
+        event.preventDefault();
+
+        const newFriend = {
+            name: this.state.name,
+            age: Number(this.state.age),
+            email: this.state.email,
+        }
+
+        this.props.add(newFriend);
+        this.setState({
+            name: '',
+            age: '',
+            email: '',
+        })
+        event.target.reset();
```

Comment:
do you need both of these (`this.setState` & `event.target.reset`) to reset your form? 🤔